financial-connections/src/main/java/com/stripe/android/financialconnections/model/FinancialConnectionsGenericInfoScreen.kt

```diff
@@ -0,0 +1,144 @@
+) : Parcelable {
+
+    @Parcelize
+    @Serializable
+    internal data class Header(
+        val title: String? = null,
+        val subtitle: String? = null,
+        val icon: Image? = null,
+        val alignment: Alignment? = null
+    ) : Parcelable
+
+    @Parcelize
+    @Serializable
+    internal data class Body(
+        val entries: List<BodyEntry>
+    ) : Parcelable {
+
+        @Serializable(with = BodyEntrySerializer::class)
+        sealed class BodyEntry : Parcelable {
+            abstract val id: String
+
+            @Parcelize
+            @Serializable
+            internal data class BodyText(
+                override val id: String,
+                val text: String,
+                val alignment: Alignment? = null,
+                val size: Size? = null
+            ) : BodyEntry()
+
+            @Parcelize
+            @Serializable
+            internal data class BodyImage(
+                override val id: String,
+                val image: Image,
+                val alt: String
+            ) : BodyEntry()
+
+            @Parcelize
+            @Serializable
+            internal data class BodyBullets(
+                override val id: String,
+                val bullets: List<GenericBulletPoint>
+            ) : BodyEntry() {
+                @Parcelize
+                @Serializable
+                internal data class GenericBulletPoint(
+                    val id: String,
+                    val icon: Image? = null,
+                    val title: String? = null,
+                    val content: String? = null
+                ) : Parcelable
+            }
+
+            @Serializable
+            @Parcelize
+            internal data class Unknown(
+                override val id: String
+            ) : BodyEntry()
+
+            // TODO@carlosmuvi: Add missing body items: prepane, forms.
+        }
+    }
+
+    @Parcelize
+    @Serializable
+    internal data class Footer(
+        val disclaimer: String? = null,
+        @SerialName("primary_cta")
+        val primaryCta: GenericInfoAction? = null,
+        @SerialName("secondary_cta")
+        val secondaryCta: GenericInfoAction? = null,
+        @SerialName("below_cta")
+        val belowCta: GenericInfoAction? = null
+    ) : Parcelable {
+        @Parcelize
+        @Serializable
+        internal data class GenericInfoAction(
+            val id: String,
+            val label: String,
+            val icon: Image? = null,
+            val action: String? = null,
+            @SerialName("test_id")
+            val testId: String? = null
+        ) : Parcelable
+    }
+
+    @Parcelize
+    @Serializable
+    internal data class Options(
+        @SerialName("full_width_content")
+        val fullWidthContent: Boolean? = null,
+        @SerialName("vertical_alignment")
+        val verticalAlignment: VerticalAlignment? = null
+    ) : Parcelable
+}
+
+@Serializable
+internal enum class Alignment {
+    @SerialName("left")
+    LEFT,
```

Comment:
Nit: Can we use pascal case for enums?

Reply:
https://github.com/stripe/stripe-android/pull/8721/commits/83be9adf84fafa508f1f33983f3a154d9418c086